pi-manage

```diff
@@ -1243,11 +1234,20 @@ def createtoken(role, days, realm, username):
 
 if __name__ == '__main__':
     # We add one blank line, to separate the messages from the initialization
-    print("""
+    print_header = True
+    try:
+        if sys.argv[1] in [ '-q', '--quiet' ]:
+            print_header = False
+            sys.argv = sys.argv[1:]
+    except IndexError:
```

Comment:
It turns out there is a method in flask_script ``Manager.add_option`` which would do the job. However, it is tricky to use and might not easily work in our code.
However, if you add the ``add_option`` anyways, we at least get a help info:

You can do this at the beginning of the file after

    manager = Manager(app)

there you can call 

    manager.add_option("-q", "--quiet", help="Do not show the privacyIDEA banner.", action="store_true")